experiments/functional/snapshot_deletion/jiva/snapshot_delete_verify_test.yml

```diff
@@ -5,18 +5,10 @@
     executable: /bin/bash
   register: controller_pod
 
-- name: List snapshots
+- name: Getting number of snapshots
   shell: >
-    kubectl exec -it {{ controller_pod.stdout }} -n {{ app_ns }} -- jivactl snapshot ls | grep -v ID | tail -n +2
+    kubectl exec -it {{ controller_pod.stdout }} -n {{ app_ns }} -- jivactl snapshot ls | grep -v ID | wc -l
   args:
     executable: /bin/bash
-  register: snapshot_list
-
-- name: Delete snapshots
-  shell: >
-    kubectl exec -it {{ controller_pod.stdout }} -n {{ app_ns }} -- jivactl snapshot rm {{item}}
-  args:
-    executable: /bin/bash
-  register: snapshot_list
-  with_items: "{{ snapshot_list.stdout_lines }}"
-
+  register: snapshot_number
+  failed_when: snapshot_number.stdout|int > 11
```

Comment:
@shashank855 instead of failing the task wait for some time until the snapshot has been deleted

Reply:
@nsathyaseelan I made the change.